commands/new_function_test.go

```diff
@@ -255,20 +257,18 @@ func Test_duplicateFunctionName(t *testing.T) {
 func writeFunctionYmlFile(functionName string, lang string, t *testing.T) {
```

Comment:
It's unclear to me why this exists in the tests. It's repeating/duplicating the code from the package. If a file has to be written to disk it should be done statically or using the method from the package.